.env.example

```
@@ -1,11 +1,17 @@
 # PostgreSQL
-  POSTGRES_USER=vow_pact
-  POSTGRES_PASSWORD=
-  POSTGRES_DB=vow_pact_development
+POSTGRES_USER=vow_pact
+POSTGRES_PASSWORD=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The POSTGRES_PASSWORD key should go before the POSTGRES_USER key

(UnorderedKey)